index.html

```diff
@@ -1009,6 +1009,25 @@ <h2>
 
         controller.abort();
       </pre>
+      <p>
+        In this example, all active screen wake locks are being aborted.
+      </p>
+      <p>
+        If the wake lock is active or pending activation,
+        <a data-link-for="WakeLock">request()</a> returns the same {{Promise}},
+        allowing us to abort any active wake lock by using the fact that
+        a |signal:AbortSignal| with |signal|.|aborted:boolean| set to `true`,
```

Comment:
```suggestion
        a |signal:AbortSignal| with |signal|.<a>aborted</a> set to `true`,
```

Reply:
Remember |foo| makes a var element. |foo|.|var| doesn’t make any sense. Soon we will have syntax for |foo.bar|, but that’s not supported yet.

Reply:
why not {{aborted:boolean}} then?